Bottom: 50c4efe374f8301ea305c2967fff3def581d56e3
Top:    def25ce7515dd36b9e3d14344a7bafe93f9e6787
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 14:34:06 +0000

hid/gtk (GL): I think the polygon renderer works in mask mode now


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 877ccaa..e9677a3 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1395,11 +1395,8 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
-  common_fill_pcb_polygon (out->fgGC, &polygon, screen);
+  gui->graphics->fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_FreeContours (&polygon.NoHoles);
-  /* THE GL fill_pcb_polygon doesn't work whilst masking */
-//  gui->fill_pcb_polygon (out->fgGC, &polygon, screen);
-//  gui->graphics->fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
   ghid_set_alpha_mult (out->fgGC, 1.0);
 
   gui->graphics->use_mask (HID_MASK_OFF);
